lambdas/process-async-cri-credential/src/main/java/uk/gov/di/ipv/core/processasynccricredential/ProcessAsyncCriCredentialHandler.java

```diff
@@ -224,51 +228,47 @@ private void processSuccessAsyncCriResponse(SuccessAsyncCriResponse successAsync
-            throw new AsyncVerifiableCredentialException(UNEXPECTED_ASYNC_VERIFIABLE_CREDENTIAL);
-        }
+    private Optional<CriResponseItem> getCriResponseItem(String userId, String state) {
+        final List<CriResponseItem> criResponseItems =
+                criResponseService.getCriResponseItemsByUserId(userId);
+        return criResponseItems.stream()
+                .filter(item -> Objects.equals(item.getOauthState(), state))
+                .findFirst();
     }
 
     private void sendIpvVcReceivedAuditEvent(
             AuditEventUser auditEventUser,
             VerifiableCredential verifiableCredential,
+            Cri cri,
             boolean isSuccessful)
             throws UnrecognisedVotException {
-        AuditEvent auditEvent =
-                AuditEvent.createWithoutDeviceInformation(
-                        AuditEventTypes.IPV_F2F_CRI_VC_RECEIVED,
-                        configService.getParameter(ConfigurationVariable.COMPONENT_ID),
-                        auditEventUser,
-                        getExtensionsForAudit(verifiableCredential, isSuccessful));
-        auditService.sendAuditEvent(auditEvent);
+        if (Cri.F2F.equals(cri)) {
+            AuditEvent auditEvent =
+                    AuditEvent.createWithoutDeviceInformation(
+                            AuditEventTypes.IPV_F2F_CRI_VC_RECEIVED,
+                            configService.getParameter(ConfigurationVariable.COMPONENT_ID),
+                            auditEventUser,
+                            getExtensionsForAudit(verifiableCredential, isSuccessful));
+            auditService.sendAuditEvent(auditEvent);
+        }
     }
 
-    void sendIpvVcConsumedAuditEvent(AuditEventUser auditEventUser, VerifiableCredential vc) {
-        AuditEvent auditEvent =
-                AuditEvent.createWithoutDeviceInformation(
-                        AuditEventTypes.IPV_F2F_CRI_VC_CONSUMED,
-                        configService.getParameter(ConfigurationVariable.COMPONENT_ID),
-                        auditEventUser,
-                        null,
-                        getRestrictedAuditDataForF2F(vc));
-        auditService.sendAuditEvent(auditEvent);
+    void sendIpvVcConsumedAuditEvent(
+            AuditEventUser auditEventUser, VerifiableCredential vc, Cri cri) {
+        if (Cri.F2F.equals(cri)) {
+            AuditEvent auditEvent =
+                    AuditEvent.createWithoutDeviceInformation(
+                            AuditEventTypes.IPV_F2F_CRI_VC_CONSUMED,
+                            configService.getParameter(ConfigurationVariable.COMPONENT_ID),
+                            auditEventUser,
+                            null,
+                            getRestrictedAuditDataForF2F(vc));
+            auditService.sendAuditEvent(auditEvent);
+        }
     }
 
     private void sendIpvVcErrorAuditEvent(ErrorAsyncCriResponse errorAsyncCriResponse) {
```

Comment:
to be consistent it might be clearer to do the same here (ie. move the conditional inside the method), for now?